src/components/pages/PageProduct/PageProduct.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import Product from "components/pages/PageProduct/components/Product";
+import Button from '@material-ui/core/Button';
+import { useHistory } from 'react-router-dom';
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    padding: theme.spacing(3, 0, 3),
+  },
+}));
+
+export default function PageProduct() {
+  const classes = useStyles();
+  const history = useHistory();
+
+  return (
+    <div className={classes.content}>
+      <Button onClick={() => history.goBack()}>Back</Button>
```

Comment:
Imagine a situation when someone sends you a link with details for specific product. You paste this link and go directly to details page. When you click "Back" you will not go to products menu but instead to the page that was open before this.

p.s. this comment is not related to the home task but it's just for your interest =)